option/gameplay: fix PDA UI opening too eagerly

rr- opened the pull request:
#### Checklist

- [x] I have read the [coding conventions](https://github.com/LostArtefacts/TRX/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes, or it is an internal change
- [x] I have added a readme entry about my new feature or OG bug fix, or it is a different change

#### Description

Resolves #3583.

rr- committed:
option/gameplay: fix PDA UI opening too eagerly

Resolves #3583.

aredfan commented:
I found a small bug, if the PDA is selected when it's facing away from the screen, the menu will appear twice.  I can made a video if needed.

rr- commented:
@aredfan added a fix but it has the potential of causing regressions with regard to other item UIs (including examine item). (I'm marking the UI as busy if the inventory item hasn't finished rotating in the Y axis, whereas it used to claim it's not busy.)

aredfan commented:
@rr- if I select a object in the menu, it'll move into position but the UI doesn't appear.  I'm also unable to deselect the object after selecting it.

<details><summary>The passport selected when facing forward</summary>

![20250912_120951_Level_0](https://github.com/user-attachments/assets/9b7d12ec-438c-4f74-9531-658968a5c638)

</details>

<details><summary>The passport selected when facing behind</summary>

![20250912_121000_Level_0](https://github.com/user-attachments/assets/b71aadfa-cacf-4dfc-9908-b897606ad017)

</details>

rr- commented:
Reverted the Y axis thing and changed approach for the settings alone. The fix now seems to address the original ticket. But if the PDA is facing away from the player, it will open too soon. However this behavior is inline with other inventory items.

aredfan reviewed (approved):
The PDA is working as expected. Thank you. 👍
